### PR TITLE
Initial Guide to Fork to Pull Request guide

### PR DESCRIPTION
**Pull request by TheDoubleH**
Added Content to the Pull Request guide

### CODE REVIEW

**Review by waldo1001 — APPROVED** [review]

File: content/Contributing/ForkAndPR/index.md [hunk 1]
```diff
@@ -7,9 +7,54 @@ weight = 100
 
 ## Abstract
 
-(coming soon)
+AL Guidelines is a community project, and as such YOU are encouraged to submit corrections and new ideas. In order to get your content included, you must submit a pull request to the GitHub Repository (Located here: https://github.com/microsoft/alguidelines). All Pull Requests are subject to approval by a minimum three admins.
```
Comment (SimonOfHH) [inline content/Contributing/ForkAndPR/index.md:10]: "by a minimum [**of**] three admins"

File: content/Contributing/ForkAndPR/index.md [hunk 2]
```diff
@@ -7,9 +7,54 @@ weight = 100
...
+
+https://docs.github.com/en/get-started/quickstart/contributing-to-projects
+{{% /notice %}}
+
 ### Step 1: Fork
+In order to work on the repository, You must [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the repository.
+
+By forking the repository, You essentially create a copy into Your own account. 
+
+Start by going to the GitHub Repository (), and press the Fork ![Fork Button](./fork_button.jpg "Fork")
```
Comment (SimonOfHH) [inline content/Contributing/ForkAndPR/index.md:34]: Missing link behind "GitHub Repository"?

File: content/Contributing/ForkAndPR/index.md [hunk 3]
```diff
@@ -7,9 +7,54 @@ weight = 100
...
+
+Once You have successfully forked the repository, go to your own GitHub repository : ![Forked Repository Representation](./ForkedRepro.png "Forked repository")
+
+You are now able to clone your own repository to your local pc and start editing using your favorite editor. [Visual Studio Code](https://code.visualstudio.com/) is perfectly fine for this task.
```
Comment (SimonOfHH) [inline content/Contributing/ForkAndPR/index.md:38]: I know VS Code is ubiquitous for any developer, But maybe there are people out there who like to proof-read and don't have VS Code available or would need to set it up on the machine they are using (I know, might be not so likely, but anyway).
Github supports basicall VS Code in the browser, by going to the repo and pressing "." - which will open VS Code directly in the tab. So there they could do editing directly in the browser. MAybe it's worth adding a (small) paragraph for that?

Comment (TheDoubleH) [reply content/Contributing/ForkAndPR/index.md:38]: That will be added later. This is the initial version of this page - I am sure a lot more information will be added

**Pull request merged**